tests/unit-tests/k8sChaosExperiments/models/chaosExperimentsManager-test.js

```diff
@@ -424,4 +428,139 @@ describe('Chaos experiments manager tests', function () {
+                jobId,
+                experiment_id: firstExId,
+                start_time: Date.now()
+            }, {
+
+                jobId,
+                experiment_id: secondExId,
+                start_time: Date.now()
+            }, {
+
+                jobId,
+                experiment_id: thirdExId,
+                start_time: Date.now() + 10000
+            }
+            ]);
+            getChaosExperimentsByIdsStub.resolves([
+                firstExperiment,
+                secondExperiment
+            ]);
+            await manager.stopJobExperimentsByJobId(jobId);
+            sinon.assert.calledOnce(getChaosJobExperimentsByJobIdStub);
+            sinon.assert.calledWith(getChaosJobExperimentsByJobIdStub, jobId);
+            sinon.assert.calledOnce(getChaosExperimentsByIdsStub);
+            sinon.assert.calledWith(getChaosExperimentsByIdsStub, [firstExId, secondExId]);
+            sinon.assert.calledOnce(deleteAllResourcesOfKindAndJobStub);
+            sinon.assert.calledWith(deleteAllResourcesOfKindAndJobStub, 'PodChaos', 'apps', jobId);
+        });
+        it('getChaosJobExperimentsByJobId throws - should stop flow and not throw', async () => {
+            const jobId = uuid();
+            getChaosJobExperimentsByJobIdStub.rejects();
+            await manager.stopJobExperimentsByJobId(jobId);
+            // not throwing
+            sinon.assert.calledOnce(getChaosJobExperimentsByJobIdStub);
+            sinon.assert.calledWith(getChaosJobExperimentsByJobIdStub, jobId);
+            sinon.assert.notCalled(getChaosExperimentsByIdsStub);
+            sinon.assert.notCalled(deleteAllResourcesOfKindAndJobStub);
+        });
+        it('getChaosExperimentsByIds throws - should stop flow and not throw', async () => {
+            getChaosJobExperimentsByJobIdStub.resolves([{
+                jobId,
+                experiment_id: firstExId,
+                start_time: Date.now()
+            }, {
+
+                jobId,
+                experiment_id: secondExId,
+                start_time: Date.now()
+            }, {
+
+                jobId,
+                experiment_id: thirdExId,
+                start_time: Date.now() + 10000
+            }
+            ]);
+            getChaosExperimentsByIdsStub.rejects();
+            await manager.stopJobExperimentsByJobId(jobId);
+            // not throwing
+            sinon.assert.calledOnce(getChaosJobExperimentsByJobIdStub);
+            sinon.assert.calledWith(getChaosJobExperimentsByJobIdStub, jobId);
+            sinon.assert.calledOnce(getChaosExperimentsByIdsStub);
+            sinon.assert.calledWith(getChaosExperimentsByIdsStub, [firstExId, secondExId]);
+            sinon.assert.notCalled(deleteAllResourcesOfKindAndJobStub);
+        });
+        it('deleteAllResourcesOfKindAndJob throws - should stop flow and not throw', async () => {
+            getChaosJobExperimentsByJobIdStub.resolves([{
```

Comment:
if the endpoint of deleteAllResourcesOfKindAndJob returns status code different from 2xx, is it throwing an error?
or resolving with response.
asking because if user gets 403/404 I don't want to fail the run